apps/src/lab2/introjs.scss

```diff
@@ -10,11 +10,27 @@
     visibility: visible;
     background-color: #fff;
     min-width: 250px;
-    max-width: 350px;
+    max-width: 650px;
     border-radius: 5px;
     box-shadow: 0 3px 30px #21212140;
     transition: opacity .1s ease-out;
   }
+
+  .introjs-tooltipReferenceLayer {
```

Comment:
This Safari (and iOS) bug was reported on GH last year (Oct 2024): https://github.com/usablica/intro.js/issues/2081.

As suggested from ticket, I added this update and it fixed the issue.

I found that this CSS fix prevents highlighted elements from being interactive so had to add a listener to the document to translate clicks and keyboard presses to the introJS overlay.

Reply:
Can you link their ticket in the comment?

Reply:
Sure - updated comment with link. Thanks!